tests/basic/backend.yaml

```diff
@@ -13,6 +13,8 @@ modules:
     registry: local
   users:
     registry: local
+    config:
+      maxPfpBytes: 1048576 # 1 MiB
```

Comment:
expand pfp

Reply:
this is inconsistent with the uploads module's size format. need to choose one and stick with it.